crates/extensions/c8y_log_manager/src/actor.rs

```diff
@@ -454,3 +404,407 @@ impl TryIntoOperationStatusMessage for LogfileRequest {
         .to_smartrest()
     }
 }
+
+#[cfg(test)]
```

Comment:
Optional: I also generally prefer keeping unit tests in the same module as the code, but considering the huge size of this file, it might be better to move these tests to a dedicated `tests.rs` file.